src/common/ui-components/CategoryNavigation/CategoryNavigation.scss

```diff
@@ -0,0 +1,71 @@
+.categoryNavigationSlider {
```

Comment:
I assume this is copy-pasted from somewhere? 
Could you add url and reference in the beginning of the file?
Just to keep the metal model of what is borrowed and what is ours?

Reply:
It's modified from the "poc" code. No references there either.